src/app/users/utils.js

```diff
@@ -358,9 +373,9 @@ const waitForIndexToUpdate = async (uid, updatedCheck) => {
 
 const mapRole = (roleId) => {
   if (roleId === 10000) {
-    return { id: 10000, description: 'Approver' };
+    return { id: 10000, description: "Approver" };
   }
-  return { id: 0, description: 'End user' };
+  return { id: 0, description: "End user" };
```

Comment:
Why are the quotes being changed to double quotes here? I think our informal standard is single quotes.

Reply:
Changed back, I think i had prettier formatting the file I was working in at one point and that might formatted on save.